### ISSUE TITLE
Yardforce NX100i

### ISSUE THREAD

**Issue by Strambauer** [cloudn1ne/Mowgli#19]
Hello,
I am going to use Mowgli on my Yardforce NX100i.
The NX100i has a GForce RM-MB-V6.0.0 Mainboard so Mowgli will work (I have already flashed it ;-) ).

I have analysed some hardware pathes wich where not in the scematic of the 500 classic, for example the ultrasonic sensors and the perimeter signal.

During this analyse phase my robot stopt booting and I don't know why. 
I know the controler is still working, because I can flash the Mowgli stuff. 
So mayby I only destroyed the firmware while measuring.

Do somebody have an firmware backup of an NX Robot? 
Or even better the protocol to the panel which i do not have analiese yet.

If someonw want to know some details about the perimeter or the ultrasonic sensors just ask ;-)

Regards 
Georg

**Comment by Pawel-sp9pw** [comment]
Hi, I'm going to upgrade mine NX100is to mowgli in winter so i could try to backup firmware.

**Comment by yannlossouarn** [comment]
@Strambauer Did you succeed in restoring your NX100i original firmware ?

**Comment by newbyr** [comment]
Hi.   Do you know which pins follow the perimiter wire?  I want to put a sensor line on this to stop a signal on a modul thanks

**Comment by Strambauer** [comment]
> [@Strambauer](https://github.com/Strambauer) Did you succeed in restoring your NX100i original firmware ?

Yes I do succeed in flashing back the orignial firmware. My fault was, that I have caused a short circuit betwen two controler pins. After "cleaning" the pins everything workt as before. The NX100i is working again with the original configuration. 




> Hi. Do you know which pins follow the perimiter wire? I want to put a sensor line on this to stop a signal on a modul thanks

I guess I have found the perimeter pins in the circuite diagram after searching for a while. They are not marked so clear, but it sould be easy to find three ignals on the connector. If you do not finde the information I have to search through my notes. 

I dropt my mowgli projekt and bought a off the shelf Navimow i108e becaus lag of time.
